### ISSUE TITLE
Return type of function doesn't match the variable type assigned while calling the function

### ISSUE THREAD

**Issue by skaruparthi** [ros-industrial-consortium/trajopt_ros#168]
In "trajopt_sco/src/gurobi_interface.cpp"
IntVec inds = vars2indsI(vars);

vars is of type sco::SizeTVec which is std::vector of type "long unsigned int"
IntVec is of type sco::IntVec which is std::vector of type "int"

This is causing the build to fail when using Gurobi.
Trying to install on Ubuntu 18.04 with Gurobi 9.0 and Latest version of Trajopt

**Comment by Levi-Armstrong** [comment]
Sorry, I do not have a license to Gurobi to confirm. If you created a PR to fix the issue we will get it merged into master.

**Comment by skaruparthi** [comment]
I have made a couple of changes and got it working. How do I go about submitting a pull request? I tried the normal way but it just said permission denied. Sorry if it is a stupid question. Thanks in advance

**Comment by Levi-Armstrong** [comment]
I would fork this repository and push your changes to your fork of this repository. Once that is done you then can create a PR from your fork to this repository.

**Comment by Levi-Armstrong** [comment]
@skaruparthi Did #174 resolve your issue?
